README.md

```diff
@@ -0,0 +1,175 @@
+### Creating the DUH document
+NOTE: this tutorial was made using version 1.15.0 of DUH
+
+The DUH document is a json5 file that describes an IP block similar to IPXACT.
+The DUH json schema is defined [here](https://github.com/sifive/duh-schema).
+DUH is also a suite of tools that help users author DUH documents and generate
+useful artefacts from them. This section describes how to use some of the DUH
+tools to author a DUH document for the PIO block.
+
+First you will need to install [DUH](https://github.com/sifive/duh). To install
+DUH in your current directory run
+```bash
+npm i duh
+```
+
+This will create a `node_modules` subdirectory in your current directory with
+all the package sources. The `duh` executable along with other useful tools are
+installed in `node_modules/.bin`.
```

Comment:
mention about setting path, and whatever the node utility for setting path is.

Reply:
do you mean like `export PATH=$PATH:$install_dir/node_modules/.bin`? and what is the node utility for setting path?

Reply:
I would recommend setting `export PATH=./node_modules/.bin:$PATH`